cpp/INIReader.cpp

```diff
@@ -32,6 +32,33 @@ int INIReader::ParseError() const
     return _error;
 }
 
+string INIReader::ParseErrorMessage() const
+{
+    // If _error is positive it means it is the line number on which a parse
+    // error occurred. This could be an overlong line, that ValueHandler
+    // indicated a user defined error, an unterminated section name, or a name
+    // without a value.
+    if (_error > 0) {
+        return "parse error on line " + std::to_string(_error);
```

Comment:
Do you think we should add this?
```suggestion
        return "parse error on line " + std::to_string(_error) + "; missing ']' or '='?";
```